testing/test_package/lib/fake.dart

```diff
@@ -224,6 +221,7 @@ typedef int LotsAndLotsOfParameters(so, many, parameters, it, should, wrap,
 
 /// This class is cool!
 class Cool {
+  // ignore: missing_return
   Cool returnCool() {}
```

Comment:
Aside: why not just return `null`? In general, I'm curious why all the ignores and not just fixes?

Reply:
The test package is intended to be a bit strange, using outdated, error-prone and/or obsolete constructions in some places.   So I don't want to accidentally make the test package too "clean".

I shouldn't be doing that anywhere _outside_ testing/ though.

Reply:
Makes sense.  Thanks!